app/cdap/components/Administration/AdminConfigTabContent/SystemPrefsAccordion/index.js

```diff
@@ -185,6 +185,7 @@ export default class SystemPrefsAccordion extends Component {
           className="btn btn-secondary"
           onClick={this.togglePrefsModal}
           data-cy="edit-system-prefs-btn"
+          data-se="edit-system-prefs-btn"
```

Comment:
Should we consider a generic name for this attribute? I couldn't think of a good one other than `data-testid` maybe? open to suggestions.